ibis/backends/flink/registry.py

```diff
@@ -34,6 +34,35 @@ def extract_field_formatter(translator: ExprTranslator, op: ops.Node) -> str:
     return extract_field_formatter
 
 
+def _cast(translator: ExprTranslator, op: ops.generic.Cast) -> str:
```

Comment:
```suggestion
def _cast(translator: ExprTranslator, op: ops.Cast) -> str:
```
Nit: I feel like, in other parts of the codebase, I usually just see the non-namespaced version. But may be something better to check with one of the maintainers, what the best practice here is. (I'd also be curious, since there are inconsistencies.)